includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -28,23 +28,8 @@ public function getData() {
 		global $wgUser;
 
 		$data = [
-			'logo' => [
-				'header' => [
-					'type' => 'link-image',
-					'href' => $this->getHref( 'fandom-logo' ),
-					// 'image' is deprecated use 'image-data' instead
-					'image' => 'wds-company-logo-fandom-powered-by-wikia',
-					'image-data' => [
-						'type' => 'wds-svg',
-						'name' => 'wds-company-logo-fandom-powered-by-wikia',
-					],
-					'title' => [
-						'type' => 'text',
-						'value' => 'Fandom powered by Wikia'
-					],
-					'tracking_label' => 'logo',
-				]
-			],
+			'logo' => $this->getLogo(),
+			'logo_small' => $this->getLogoSmall(),
```

Comment:
I'm not fully convinced to this direction.

So far all sections defined at this level were supposed to be rendered by client. Right now we're introducing multiple sections representing the same component on a page - which is logo on different screens.

IMO: It's more intuitive to have all `logo`, `logo_small` and `logo_small_tagline` in one section. It makes fact that they're alternatives more visible.
Additionally I'd make `logo_small_tagline` part of `logo_small` and include it only when it's present.

I understand the cost of introducing new types in JSON schema, although I believe it makes API easier to use and understand.

Ping @idradm @sebastianmarzjan @rogatty